### PR TITLE
[ci skip] Grammar correction

### PR DESCRIPTION
**Pull request by Chris6ix**
Basically #8898 and #8899, but correct.
The hyphen of "High-performance" should just be added to "About" :)

### CODE REVIEW

**Review by Jakubk15 — APPROVED** [review]

**Review by lynxplay — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ you will most likely use this for WSL), `homebrew` (macOS / Linux), and more:
 - `git` (package `git` everywhere);
 - A Java 17 or later JDK (packages vary, use Google/DuckDuckGo/etc.).
   - [Adoptium](https://adoptium.net/) has builds for most operating systems.
-  - Paper requires JDK 17 to build, however makes use of Gradle's
+  - Paper requires JDK 17 to build, however, makes use of Gradle's
```
Comment (Machine-Maker) [inline CONTRIBUTING.md:32]: Something about this still reads wrong to me. I feel like there needs to be an `it` before `makes use of` to be correct. But please correct me if I'm wrong.

**Review by Machine-Maker — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (olijeffers0n) [reply CONTRIBUTING.md:32]: It is fine as it is

Comment (Chris6ix) [reply CONTRIBUTING.md:32]: To me, that sounds wrong. `it` would refer to `Paper`, but there is no need to refer to it again, `Paper` is already named, and the connection to `makes use of` is clear.

I double-checked with my [Language checker](https://languagetool.org/?pk_campaign=addon2-popup-logo), and it seems to agree with me.

**Review by Machine-Maker — APPROVED** [review]

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -29,7 +29,7 @@ you will most likely use this for WSL), `homebrew` (macOS / Linux), and more:
...
     [Toolchains](https://docs.gradle.org/current/userguide/toolchains.html)
     feature to allow building with only JRE 8 or later installed. (Gradle will
```
Comment (jpenilla) [inline CONTRIBUTING.md]: Can you update this to 11?

**Review by jpenilla — CHANGES_REQUESTED** [review]

**Review by MiniDigger — APPROVED** [review]

File: CONTRIBUTING.md (hunk 2 above)
Comment (Chris6ix) [reply CONTRIBUTING.md]: > Can you update this to 11?

Only this specific one, or the ones above and below too?

Comment (Chris6ix) [reply CONTRIBUTING.md]: Wait, are you talking about 17 or 8?

Comment (jpenilla) [reply CONTRIBUTING.md]: 8 -> 11
paperweight needs a minimum JRE 11 runtime now instead of 8

Comment (Chris6ix) [reply CONTRIBUTING.md]: Done :)

**Review by jpenilla — APPROVED** [review]

**Pull request merged**